application/src/main/data/upgrade/3.8.1/schema_update.sql

```diff
@@ -25,6 +25,23 @@ UPDATE user_credentials c SET failed_login_attempts = (SELECT (additional_info::
 UPDATE tb_user SET additional_info = (additional_info::jsonb - 'lastLoginTs' - 'failedLoginAttempts' - 'userCredentialsEnabled')::text
   WHERE additional_info IS NOT NULL AND additional_info != 'null';
 
+-- UPDATE RULE NODE DEBUG MODE TO DEBUG STRATEGY START
+
+ALTER TABLE rule_node ADD COLUMN IF NOT EXISTS debug_failures boolean DEFAULT false;
+ALTER TABLE rule_node ADD COLUMN IF NOT EXISTS debug_all_until bigint NOT NULL DEFAULT 0;
+DO
+$$
+    BEGIN
+        IF EXISTS (SELECT 1 FROM information_schema.columns WHERE table_name = 'rule_node' AND column_name = 'debug_mode')
+            THEN
+                UPDATE rule_node SET debug_all_until = (extract(epoch from now()) + 3600) * 1000 WHERE debug_mode = true;
```

Comment:
1 hour of debug? Why not 15 minutes? yml references DEBUG_MODE_MAX_DURATION_MINUTES which is 15 minutes.

Reply:
I think its better to set bigger value for upgrade, because we will spend some time for restart all services and customers dont know about this new feature, so they will have more time for debug. We can afford it.